scripts/check-version.js

```diff
@@ -0,0 +1,10 @@
+const semver = require('semver');
+const engines = require('../package').engines;
+
+const version = engines.node;
+if (!semver.satisfies(process.version, version)) {
```

Comment:
run `npm lint` and you'll see there are some errors

put `/*global process*/` at the top of this file to get rid of one

and then `// eslint-disable-next-line` above the `console.log` lines